### PR TITLE
Add tech preview disclaimer

### PR DESCRIPTION
**Pull request by jgu17**
Added tech preview disclaimer on the welcome page.

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -1,9 +1,33 @@
...
+use. They are mainly included for customer convenience and give customers a
+chance to test new technologies within an enterprise environment.
+
+This release deploys the following set of core OpenStack services:
+
+  * Keystone
+  * Glance
+  * Cinder
+  * Nova
+  * Nuetron
```
Comment (jesusaurus) [inline doc/source/index.rst]: typo: Neutron

File: doc/source/index.rst [hunk 2]
```diff
@@ -1,9 +1,33 @@
...
+  * Heat
+  * Horizon
+
+Whether it will be moved to a fully supported product later, depends on customer
```
Comment (jesusaurus) [inline doc/source/index.rst]: extraneous comma

File: doc/source/index.rst [hunk 3]
```diff
@@ -1,9 +1,33 @@
...
+and market feedback. A technology preview does not automatically result in
+support at a later point in time. It could be dropped at any time and SUSE is not
+committed to provide a technology preview later in the product cycle.
```
Comment (jesusaurus) [inline doc/source/index.rst]: *providing

File: doc/source/index.rst (hunk 2 above)
Comment (jgu17) [reply doc/source/index.rst]: I debated about it, but it is how it was in the soc7 doc where I stole the language. I agree with you.

File: doc/source/index.rst (hunk 1 above)
Comment (jgu17) [reply doc/source/index.rst]: hahaha... I sensed somethng was wrong, and checked it out twice and though I spelled it right :-).

File: doc/source/index.rst (hunk 3 above)
Comment (jgu17) [reply doc/source/index.rst]: good catch, again, a legacy error from soc7 doc, lmao.... where were you Jonathan?

File: doc/source/index.rst (hunk 1 above)
Comment (jgu17) [reply doc/source/index.rst]: fixed

File: doc/source/index.rst (hunk 3 above)
Comment (jgu17) [reply doc/source/index.rst]: fixed

File: doc/source/index.rst [hunk 4]
```diff
@@ -1,9 +1,33 @@
...
+The socok8s project automates SUSE Containerized OpenStack provisioning and
+lifecycle management on SUSE Container as a Service Platform (CaaSP) and
+SUSE Enterprise Storage (SES), using Airship, OpenStack Helm, shell scripts,
+and Ansible playbooks.
+
+As a technology preview, the included features are not supported. They may be
+functionally incomplete, unstable or in other ways not suitable for production
+use. They are mainly included for customer convenience and give customers a
+chance to test new technologies within an enterprise environment.
+
```
Comment (tbosworth7) [inline doc/source/index.rst]: We need to rework the paragraph above.  

Suggested rework.

As a technology preview, this is not meant for production implementation and a typical support subscription for the technology preview is not being offered.  This technology preview gives customers and partners the ability to test and see the new technologies that will be offered with full support in future releases.  As a caution some of the functionality currently in our generally available SUSE OpenStack Cloud product will not be available in this tech preview.

File: doc/source/index.rst [hunk 5]
```diff
@@ -1,9 +1,33 @@
...
+  * Nova
+  * Neutron
+  * Heat
+  * Horizon
+
+Whether it will be moved to a fully supported product later depends on customer
+and market feedback. A technology preview does not automatically result in
+support at a later point in time. It could be dropped at any time and SUSE is not
+committed to providing a technology preview later in the product cycle.
+
```
Comment (tbosworth7) [inline doc/source/index.rst]: Suggest we rework the above paragraph as follows.

This technology preview will not be moved into a production support mode of operation. This technology is being considered for the next release of SUSE OpenStack Cloud 10.  There is a possibility of refreshing the capabilities of the technology preview later this year.

File: doc/source/index.rst [hunk 6]
```diff
@@ -1,9 +1,33 @@
...
+Please, give your SUSE representative feedback, including your experience and
+use case.
```
Comment (tbosworth7) [inline doc/source/index.rst]: Replace
Please provide your feedback using the beta mailing lists that are documented on the technology preview download page or your SUSE representative.  We are very interested in your experience and use case.

**Pull request merged**